indexd_test_utils2/indexd_settings.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from indexd.alias.drivers.alchemy import SQLAlchemyAliasDriver
+from indexd.auth.drivers.alchemy import SQLAlchemyAuthDriver
+from indexd.index.drivers.alchemy import SQLAlchemyIndexDriver
+
+
+def get_settings(pg_url):
+    CONFIG = {}
+
+    CONFIG['JSONIFY_PRETTYPRINT_REGULAR'] = False
```

Comment:
Is there a need in this module to use single quotes for string literals rather than double quotes?

Reply:
No, waiting approval of my black ticket, black should automatically fix those things.

Reply:
👍